roles/BCPC-Hadoop-Worker.json

```diff
@@ -12,11 +12,11 @@
     "recipe[bcpc-hadoop::hdp_repo]",
     "recipe[bcpc-hadoop::configs]",
     "recipe[pam::default]",
+    "recipe[bach_spark::default]",
```

Comment:
Good call on a fresh cluster, HDFS would be down to upload the JAR's and yeah no hadoop software even yet; I wonder how to keep from racing as the nodemanager needs the Spark shuffle service but we need to install that first -- the HDFS upload of JARs could be done later.